pos_cashier_select/static/description/index.html

```diff
@@ -0,0 +1,83 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan" style="color:#875A7B;">POS Cashier Select</h2>
+            <h3 class="oe_slogan">Choose a cashier before a payment</h3>
+        </div>
+    </div>
+</section>
+
```

Comment:
Add one more section that contains the text:
"The module forces to select a cashier before switching to payment screen. Once user clicks on "Payment", custom popup is shown and user can select a cashier by scanning cashier's barcode or by clicking one from the list."